src/components/Form.js

```diff
@@ -1,13 +1,18 @@
-export default function Form({ onSubmit }) {
+export default function Form({ onSubmit, uploadImage }) {
   return (
-    <form className="AddNewCake__form" onSubmit={onSubmit}>
+    <form
+      className="AddNewCake__form"
+      onSubmit={onSubmit}
+      onChange={uploadImage}
+    >
       <i className="input-file--icon"></i>
 
       <input
         type="file"
         id="cakeImage"
         name="cakeImage"
         accept="image/png, image/jpeg"
+        onChange={uploadImage}
```

Comment:
Is it necesary that you have the onChange twice ?
Wouldn't it be enough if you have either in the form or the input ?

Reply:
I see your point. You are right. In this current version I am not using the Form as a component. Will change it once I make the component working. :)